packages/dds/shared-object-base/src/types.ts

```diff
@@ -54,7 +54,7 @@ export interface ISharedObjectEvents extends IErrorEvent {
 
 /**
  * Base interface for shared objects from which other interfaces derive. Implemented by SharedObject
- * @public
+ * @alpha
```

Comment:
It's fine with me, but maybe worth doble checking that the message is - we do not support 3rd parties creating DDSs. Is that actually the case?

Reply:
I believe that is correct. @kashms can you confirm this is still the case?

Reply:
We do not support users of the declarative API doing creating DDSes. If they were going to implement a custom one, they would need to use alpha APIs (the encapsulated API) anyway since SharedObject (the class) is alpha as are a lots of things they would need to interact with (like the DeltaManager).

Reply:
Yes, @Josmithr alpha(aka /legacy export) is the right designation here for custom DDSes in FF 2